src/test/java/io/pinecone/PineconeIndexOperationClientTest.java

```diff
@@ -170,4 +170,56 @@ public void testCreateIndexWithAllFields() throws IOException {
         verify(mockClient, times(1)).newCall(any(Request.class));
         verify(mockCall, times(1)).execute();
     }
+
+    @Test
+    public void testDescribeIndex() throws IOException {
+        String indexName = "testIndex";
+        String indexJsonString = "{\"database\":{\"name\":\"testIndex\",\"metric\":\"cosine\",\"dimension\":3," +
+                "\"replicas\":1,\"shards\":1,\"pods\":1,\"pod_type\":\"p1.x1\"}," +
+                "\"status\":{\"waiting\":[],\"crashed\":[],\"host\":\"url\",\"port\":433," +
+                "\"state\":\"Ready\",\"ready\":true}}";
+        PineconeClientConfig clientConfig = new PineconeClientConfig()
+                .withApiKey("testApiKey")
+                .withEnvironment("testEnvironment");
+        IndexMetaDatabase metaDatabase = new IndexMetaDatabase()
+                .withName("testIndex")
+                .withMetric("cosine")
+                .withDimension(3)
+                .withReplicas(1)
+                .withShards(1)
+                .withPods(1)
+                .withPodType("p1.x1");
+        IndexStatus indexStatus = new IndexStatus()
+                .withHost("url")
+                .withState("Ready")
+                .withReady(true)
+                .withPort("433");
+
+
+        IndexMeta expectedIndexMeta = new IndexMeta()
+                .withMetaDatabase(metaDatabase)
+                .withStatus(indexStatus);
+
+        expectedIndexMeta.withMetaDatabase(metaDatabase);
+
```

Comment:
Are we calling `expectedIndexMeta.withMetaDatabase(metaDatabase)` twice here for a specific reason?

Reply:
No, there's no need for second call. Thanks for pointing it out.